jenkins-config/clouds/openstack/cattle/centos7-builder-1c-1g.cfg

```diff
@@ -1,3 +1,4 @@
-IMAGE_NAME=ZZCI - CentOS 7 - builder - x86_64 - 20210519-032457.062
+IMAGE_NAME=ZZCI - CentOS 7 - builder - x86_64 - 20210519-041348.372
 VOLUME_SIZE=20
 HARDWARE_ID=v2-highcpu-1
+VOLUME_SIZE=20
```

Comment:
Volume size is now listed twice. I think you should remove it from line 2.

Reply:
@MightyNerdEric This is fixed up here: https://github.com/zowe/ci-management/pull/29
Thanks for catching that.